### PR TITLE
Credits

### LINKED ISSUE #2528: Add credits for photos and icons for redesigned homepage
### Overview
We need to add credits for all new icons and photographs used in the redesigned version of the homepage so that the Credits page is up to date. 

### Note: Feature Branch
The pull request created from this issue should be pushed into a feature branch called "feature-homepage-launch." Please note this [wiki article](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch) on how to work off of a feature branch.

### Action Items

- [ ] Add credits to the Credits Page for all the icons and photographs used on the redesigned homepage. All the credit information can be found [here](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=17951%3A122724).

### Resources/Instructions
[HfLA - Credits Page](https://www.hackforla.org/credits/)
[Figma - Homepage Redesign: Credits](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=17951%3A122724)

### PR DESCRIPTION
**Pull request by jqg123**
Fixes #2528

### What changes did you make and why did you make them ?

  -new icons
  -credits
  -figma prototype to feature branch

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visuals before changes are applied</summary>

image
screencapture-localhost-4000-credits-2022-02-20-20_08_17

</details>

<details>
<summary>Visuals after changes are applied</summary>
  
image
screencapture-localhost-4000-credits-2022-02-20-20_36_00

</details>

### CODE REVIEW

**Comment by JessicaLucindaCheng** [comment]
- Availability: 1.5 hours
- Review ETA: Tue, Feb 15, 2022

**Comment by SAUMILDHANKAR** [comment]
Review ETA: 2/17
Availability: Thursday 6-8 PM

File: _data/internal/credits/discussion-teamwork.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: Discussion Teamwork
+title-link: https://thenounproject.com/kerismaker/collection/product-management-line/?i=3656728
+content: icon
+used-in: Homepage
+artist: Tezar Tantular
+provider: Noun Project
+provider-link: https://thenounproject.com/kerismaker/collection/product-management-line/?i=3656728
```
Comment (JessicaLucindaCheng) [inline _data/internal/credits/discussion-teamwork.yml]: This link should be `https://thenounproject.com/`

File: _data/internal/credits/operations.yml [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 content: icon
-used-in: Home
+used-in: Homepage
 artist: Made by Made
-provider: Noun Project 
-provider-link: https://thenounproject.com/
-image-url: /assets/images/hero/communities-icons/operations.png
-alt: 'Operations Icon'
+provider: Noun Project
+provider-link: 'https://thenounproject.com/'
```
Comment (JessicaLucindaCheng) [inline _data/internal/credits/operations.yml]: The link doesn't need single quotes around it.

File: _data/internal/credits/datascience.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+title: Data Science
+title-link: https://www.thenounproject.com/search/?q=data+science&i=1705416
```
Comment (JessicaLucindaCheng) [inline _data/internal/credits/datascience.yml]: The link should be `https://thenounproject.com/search/?q=data+science&i=1705416`

File: _data/internal/credits/operations.yml [hunk 4]
```diff
@@ -1,12 +1,12 @@
 ---
 title: Operations
-title-link: https://thenounproject.com/made.somewhere/collection/work/?i=1819014
+title-link: https://thenounproject.com/icon/operations-1819014/
```
Comment (JessicaLucindaCheng) [inline _data/internal/credits/operations.yml:3]: The link should be `https://thenounproject.com/search/?q=operations&i=1819014`

File: _data/internal/credits/datascience.yml [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+content: icon
+used-in: Homepage
+artist: Med Marki
+provider: Noun Project
+provider-link: https://www.thenounproject.com/
```
Comment (JessicaLucindaCheng) [inline _data/internal/credits/datascience.yml]: The link should be `https://thenounproject.com/`

File: _data/internal/credits/accounting.yml [hunk 6]
```diff
@@ -0,0 +1,12 @@
+---
+title: Accounting
+title-link: https://thenounproject.com/search/?q=accounting&i=3669009
+content: icon
+used-in: Homepage
+artist: Mada Creative
+provider: Noun Project
+provider-link: https://thenounproject.com/
+image-url: assets/images/accounting/noun-accounting-3669009.svg
+
```
Comment (JessicaLucindaCheng) [inline _data/internal/credits/accounting.yml]: Missing an `alt` field for alt-text

File: _data/internal/credits/hand.yml [hunk 7]
```diff
@@ -0,0 +1,12 @@
+---
+title: Hands
+title-link: https://unsplash.com/photos/LjqARJaJotc
+content: icon
+used-in: Homepage
+artist: Clay Banks
+provider: Unsplash
+provider-link: https://unsplash.com/
+image-url: /assets/images/hand/clay-banks-LjqARJaJotc-unsplash.jpg
+
```
Comment (JessicaLucindaCheng) [inline _data/internal/credits/hand.yml]: `alt` field for alt text is missing.

File: _data/internal/credits/infinity.yml [hunk 8]
```diff
@@ -0,0 +1,12 @@
+---
+title: Infinity
+title-link: https://thenounproject.com/search/?q=infinity&i=686117
+content: icon
+used-in: Homepage
+artist: Yeong Rong Kim
+provider: Noun Project
+provider-link: https://thenounproject.com/
+image-url: /assets/images/infinity/noun-infinity-686117.svg
+
```
Comment (JessicaLucindaCheng) [inline _data/internal/credits/infinity.yml]: Missing an alt field for alt text.

File: _data/internal/credits/marketing.yml [hunk 9]
```diff
@@ -1,12 +1,12 @@
...
-used-in: Home
+used-in: Homepage
 artist: IconPai
-provider: Noun Project 
+provider: Noun Project
 provider-link: https://thenounproject.com/
-image-url: /assets/images/hero/communities-icons/marketing.png
-alt: 'Marketing Icon'
+image-url: /assets/images/marketing/noun-marketing-3154527.svg
+
```
Comment (JessicaLucindaCheng) [inline _data/internal/credits/marketing.yml]: Missing an alt field for alt text.

File: _data/internal/credits/socialsafetynet.yml [hunk 10]
```diff
@@ -0,0 +1,12 @@
+---
+title: Social Safety Net
+used-in: Homepage
+artist: Austin Kehmeier
+provider: Unsplash
+title-link: https://unsplash.com/photos/lyiKExA4zQA
+content: icon
+provider-link: https://unsplash.com/
+image-url: /assets/images/socialsafetynet/austin-kehmeier-lyiKExA4zQA-unsplash.jpg
+
```
Comment (JessicaLucindaCheng) [inline _data/internal/credits/socialsafetynet.yml]: Missing an alt field for alt text.

File: _data/internal/credits/justice.yml [hunk 11]
```diff
@@ -0,0 +1,13 @@
...
+
+used in: Homepage
+artist: Sean Lee
+provider: Unsplash
+title: Justice
+title-link: https://unsplash.com/photos/bvHcG6334dE
+content: icon
+provider-link: https://unsplash.com/
+image-url: /assets/images/justice/sean-lee-bvHcG6334dE-unsplash.jpg
+
```
Comment (JessicaLucindaCheng) [inline _data/internal/credits/justice.yml]: Missing an alt field for alt text.

File: _data/internal/credits/environment.yml [hunk 12]
```diff
@@ -0,0 +1,12 @@
+---
+title: Environment
+title-link: https://unsplash.com/photos/kbTp7dBzHyY
+content: icon
+used in: Homepage
+artist: Matt Palmer
+provider: Unsplash
+provider-link: https://unsplash.com/
+image-url: /assets/images/environment/matt-palmer-kbTp7dBzHyY-unsplash.jpg
+
```
Comment (JessicaLucindaCheng) [inline _data/internal/credits/environment.yml]: Missing an alt field for alt text.

File: _data/internal/credits/diversityequityinclusion.yml [hunk 13]
```diff
@@ -0,0 +1,12 @@
+---
+title: Diversity, Equity, Inclusion
+title-link: https://burst.shopify.com/photos/team-hands-middle?q=matthew+henry+diversity
+content: icon
+Used in: Homepage
+artist: Matthew Henry
+provider: Burst
+provider-link: https://burst.shopify.com/
+image-url: /assets/images/diversityequityinclusion/team-hands-middle.jpg
+
```
Comment (JessicaLucindaCheng) [inline _data/internal/credits/diversityequityinclusion.yml]: Missing an alt field for alt text.

File: _data/internal/credits/VotingRepresentation.yml [hunk 14]
```diff
@@ -0,0 +1,13 @@
+---
+title: Voting & Representation
+used-in: Homepage
+title-link: https://unsplash.com/photos/GidItEeb_24
+content: icon
+artist: Christina-wocintechchat
+Provider: Unsplash
+provider-link: https://unsplash.com/
+image-url: /assets/images/Voting&Representation/christina-wocintechchat-com-mxOdUd1BQIM-unsplash.jpg
+
```
Comment (JessicaLucindaCheng) [inline _data/internal/credits/VotingRepresentation.yml]: Missing an alt field for alt-text.

File: _data/internal/credits/civictechinfrastructure.yml [hunk 15]
```diff
@@ -0,0 +1,12 @@
+---
+title: Civic Tech Infrastructure
+title-link: https://unsplash.com/photos/unRkg2jH1j0
+content: icon
+used-in: Homepage
+artist: Adeolu Eletu
+provider: Unsplash
+provider-link: https://unsplash.com/
+image-url: /assets/images/civictechinfrastructure/adeolu-eletu-unRkg2jH1j0-unsplash.jpg
+alt: 'Image of people together in community'
```
Comment (JessicaLucindaCheng) [inline _data/internal/credits/civictechinfrastructure.yml]: This alt text description doesn't match the image. Please use a description that describes the image.

File: _data/internal/credits/justice.yml [hunk 16]
```diff
@@ -0,0 +1,13 @@
+---
+
+used in: Homepage
+artist: Sean Lee
+provider: Unsplash
+title: Justice
```
Comment (JessicaLucindaCheng) [inline _data/internal/credits/justice.yml]: `title: Justice` should go on the line before the line with `used-in` field.

File: _data/internal/credits/justice.yml [hunk 17]
```diff
@@ -0,0 +1,13 @@
+---
+
+used in: Homepage
```
Comment (JessicaLucindaCheng) [inline _data/internal/credits/justice.yml]: Fields can't have spaces. It should be `used-in`

File: _data/internal/credits/VotingRepresentation.yml [hunk 18]
```diff
@@ -0,0 +1,13 @@
+---
+title: Voting & Representation
+used-in: Homepage
+title-link: https://unsplash.com/photos/GidItEeb_24
+content: icon
+artist: Christina-wocintechchat
+Provider: Unsplash
```
Comment (JessicaLucindaCheng) [inline _data/internal/credits/VotingRepresentation.yml]: `provider` should be lowercase.

File: _data/internal/credits/environment.yml [hunk 19]
```diff
@@ -0,0 +1,12 @@
+---
+title: Environment
+title-link: https://unsplash.com/photos/kbTp7dBzHyY
+content: icon
+used in: Homepage
```
Comment (JessicaLucindaCheng) [inline _data/internal/credits/environment.yml]: Should be `used-in`

File: _data/internal/credits/diversityequityinclusion.yml [hunk 20]
```diff
@@ -0,0 +1,12 @@
+---
+title: Diversity, Equity, Inclusion
+title-link: https://burst.shopify.com/photos/team-hands-middle?q=matthew+henry+diversity
+content: icon
+Used in: Homepage
```
Comment (JessicaLucindaCheng) [inline _data/internal/credits/diversityequityinclusion.yml]: Should be `used-in`

File: _data/internal/credits/VotingRepresentation.yml [hunk 21]
```diff
@@ -0,0 +1,13 @@
+---
+title: Voting & Representation
+used-in: Homepage
+title-link: https://unsplash.com/photos/GidItEeb_24
+content: icon
+artist: Christina-wocintechchat
```
Comment (JessicaLucindaCheng) [inline _data/internal/credits/VotingRepresentation.yml]: The artist should be `Phillip Goldsberry`

**Review by JessicaLucindaCheng — DISMISSED** [review]
@jqg123 In addition to my inline comments for changes, please make these changes too.

- [ ] There might have been possible changes to the branch since you last worked on this issue. Check the last pr merged on to the feature-homepage-launch branch, which should be a pr by arghmatey. In that pr, click the "Files Changed" to see what files were added.
  - [ ] Specifically, you should see what images were already added by comparing the images in Figma. 
    - [ ] There are some svg images, such as infinite.svg, that do not show up as an image but as code when looking in the "Files Changed" tab, so you will have to use Docker by using 'docker-compose up' to see those images.
    - [ ] Then make a note of what the file names are for those images and edit your credits files' `image-url` fields accordingly with those image file names.
  - [ ] Delete any images that you added that are already on the feature-homepage-launch branch. To see what images you added, check the ["Files Changed" tab for your pr](https://github.com/hackforla/website/pull/2774/files).
    - [ ] I think the only image you need to keep is this one with hands:
        
      Hands-image-to-keep  

- [ ] The citizen-engagement.yml should not have been edited as it is a credit file for a different image on a different page. Please revert the changes for that file and create a new file called citizen-engagement-2.yml
  - [ ] Change the alt field text description to match the image.

- [ ] The design.yml should not have been edited as it is a credit file for a different image on a different page. Please revert the changes for that file and create a new file called design-2.yml
  - [ ] title-link should be `https://thenounproject.com/icon/design-3160558/`

- [ ] discussion-teamwork.yml is missing an alt field for alt text.

- [ ] In the section under "What changes did you make and why did you make them?" in your original post/comment, you wrote "figma prototype to feature branch". I'm confused what that means. Can you please clarify what you meant by that in your original post/comment?

- [ ] I don't see an image for the section "Visuals before changes are applied" in your original post/comment. Please edit your comment/post with a before image. You can use [GoFullPage Chrome Extension](https://chrome.google.com/webstore/detail/gofullpage-full-page-scre/fdpohaocaechififmbbbbbknoalclacl?hl=en) to get a full page screenshot.

- [ ] Also, I can't see all your images in the "Visuals after changes are applied". Please add a blank line in between each image in your original post/comment.

- [ ] Please add your pr to the Project Board.

  <details>
  
    <summary>Click here for more detailed instructions on how to add your pr to the Project Board</summary>
    
    1) On the right, under the "Projects" section
    how-to-add-to-the-project-board1
    
    2) Click on the gear icon
    Add-to-project-board
    
    3) Then click on Project Board to add your pull request to the project board.
    how-to-add-to-the-project-board3
  
  </details>
  
 
- [ ] When you are ready for me to re-review your pull request, please click the following in the Reviewers section on the right:
image

File: _data/internal/credits/VotingRepresentation.yml [hunk 22]
```diff
@@ -0,0 +1,13 @@
...
+Provider: Unsplash
+provider-link: https://unsplash.com/
+image-url: /assets/images/Voting&Representation/christina-wocintechchat-com-mxOdUd1BQIM-unsplash.jpg
```
Comment (JessicaLucindaCheng) [inline _data/internal/credits/VotingRepresentation.yml]: The image-url link should be `/assets/images/program-areas/vote-representation.png`

**Comment by SAUMILDHANKAR** [comment]
Revised review ETA: 5 PM 2/23/22
Availability: Wednesday 3-5 PM Pacific Time

File: assets/images/discussion-teamwork/noun-discussion-teamwork-3656728.svg [hunk 23]
```diff
@@ -0,0 +1,34 @@
...
+  <path d="m373.33 338.33c-6.4414 0-11.664 5.2266-11.664 11.668s5.2227 11.668 11.664 11.668c6.4453 0 11.668-5.2266 11.668-11.668s-5.2227-11.668-11.668-11.668z"/>
+  <path d="m420 338.33c-6.4414 0-11.668 5.2266-11.668 11.668s5.2266 11.668 11.668 11.668 11.668-5.2266 11.668-11.668-5.2266-11.668-11.668-11.668z"/>
+  <use x="70" y="735" xlink:href="#d"/>
+  <use x="220.503906" y="735" xlink:href="#a"/>
+  <use x="287.953125" y="735" xlink:href="#c"/>
+  <use x="350.015625" y="735" xlink:href="#b"/>
+  <use x="450.105469" y="735" xlink:href="#a"/>
+  <use x="639.945312" y="735" xlink:href="#a"/>
+ </g>
+</svg>
```
Comment (SAUMILDHANKAR) [inline assets/images/discussion-teamwork/noun-discussion-teamwork-3656728.svg:34]: This image is already there in the feature-homepage-launch branch, so please remove it.

File: _data/internal/credits/discussion-teamwork.yml [hunk 24]
```diff
@@ -0,0 +1,11 @@
...
+title: Discussion Teamwork
+title-link: https://thenounproject.com/kerismaker/collection/product-management-line/?i=3656728
+content: icon
+used-in: Homepage
+artist: Tezar Tantular
+provider: Noun Project
+provider-link: https://thenounproject.com/
+image-url: /assets/images/discussion-teamwork/noun-discussion-teamwork-3656728.svg
+type: icon
+---
```
Comment (SAUMILDHANKAR) [inline _data/internal/credits/discussion-teamwork.yml:11]: Your yml file should basically edit the following yml file already in the feature branch as per new details on the Figma page. [_data/internal/credits/product-management.yml](https://github.com/hackforla/website/pull/2373/files#diff-5d43d5d9daf112bf202b7bee4f1bfe98b81e0f85b7441f5890289d85f544ff07)

File: _data/internal/credits/WorkforceDevelopment.yml [hunk 25]
```diff
@@ -0,0 +1,13 @@
...
+title-link: https://unsplash.com/photos/GidItEeb_24
+content: icon
+artist: Christina-wocintechchat
+provider: Unsplash
+provider-link: https://unsplash.com/
+image-url: /assets/images/WorkforceDevelopment/christina-wocintechchat-com-mxOdUd1BQIM-unsplash.jpg
+alt: 'Picture of them looking at computer'
+type: icon
+---
+
```
Comment (SAUMILDHANKAR) [inline _data/internal/credits/WorkforceDevelopment.yml:13]: Please remove this yml file as this is not there on the Figma page.

File: _data/internal/credits/design.yml [hunk 26]
```diff
@@ -1,12 +1,12 @@
...
-image-url: /assets/images/credits/credits-hero.svg
-alt: 'Design Community Concept Illustration'
+used-in: Homepage
+artist: Eucalyp
+provider: Noun Project
+provider-link: 'https://www.thenounproject.com/'
+image-url: /assets/images/design/noun-design-3160558.svg
+alt: 'Design'
 type: icon
 ---
```
Comment (SAUMILDHANKAR) [inline _data/internal/credits/design.yml:12]: There should be no changes made to this file, i.e., it should look exactly as it currently is on the feature branch and hence should not be in the files changed tab.

File: _data/internal/credits/design-2.yml [hunk 27]
```diff
@@ -0,0 +1,12 @@
+---
+title: Design
+title-link: https://www.freepik.com/free-vector/design-community-concept-illustration_6610841.htm
+content: icon
+used-in: Credits
```
Comment (SAUMILDHANKAR) [inline _data/internal/credits/design-2.yml:5]: Please change Credits to Homepage

File: _data/internal/credits/design-2.yml [hunk 28]
```diff
@@ -0,0 +1,12 @@
...
+artist: stories
```
Comment (SAUMILDHANKAR) [inline _data/internal/credits/design-2.yml:6]: Please change stories to Eucalyp

File: _data/internal/credits/design-2.yml [hunk 29]
```diff
@@ -0,0 +1,12 @@
...
+provider: Freepik
```
Comment (SAUMILDHANKAR) [inline _data/internal/credits/design-2.yml:7]: Please change Freepik to Noun Project

File: _data/internal/credits/design-2.yml [hunk 30]
```diff
@@ -0,0 +1,12 @@
...
+provider-link: 'https://www.freepik.com/'
```
Comment (SAUMILDHANKAR) [inline _data/internal/credits/design-2.yml:8]: Please use the following link: 'https://www.thenounproject.com/'

File: _data/internal/credits/design-2.yml [hunk 31]
```diff
@@ -0,0 +1,12 @@
...
+image-url: /assets/images/credits/credits-hero.svg
+alt: 'Design Community Concept Illustration'
```
Comment (SAUMILDHANKAR) [inline _data/internal/credits/design-2.yml:10]: Please remove this line.

File: assets/images/design/noun-design-3160558.svg [hunk 32]
```diff
@@ -0,0 +1,64 @@
...
+  <use x="311.328125" y="691.25" xlink:href="#a"/>
+  <use x="355.078125" y="691.25" xlink:href="#d"/>
+  <use x="384.964844" y="691.25" xlink:href="#l"/>
+  <use x="416.113281" y="691.25" xlink:href="#k"/>
+  <use x="442.046875" y="691.25" xlink:href="#b"/>
+  <use x="471.570312" y="691.25" xlink:href="#j"/>
+  <use x="486.566406" y="691.25" xlink:href="#a"/>
+  <use x="515.082031" y="691.25" xlink:href="#i"/>
+ </g>
+</svg>
```
Comment (SAUMILDHANKAR) [inline assets/images/design/noun-design-3160558.svg:64]: Please use the credits folder [website/assets/images/credits/] for this image.

File: _data/internal/credits/design-2.yml [hunk 33]
```diff
@@ -0,0 +1,12 @@
+---
+title: Design
+title-link: https://www.freepik.com/free-vector/design-community-concept-illustration_6610841.htm
+content: icon
+used-in: Credits
+artist: stories
+provider: Freepik
+provider-link: 'https://www.freepik.com/'
+image-url: /assets/images/credits/credits-hero.svg
```
Comment (SAUMILDHANKAR) [inline _data/internal/credits/design-2.yml:9]: image-url should be changed based on where the image is stored. Please see my comment below next to this file: assets/images/design/noun-design-3160558.svg

File: _data/internal/credits/design-2.yml [hunk 34]
```diff
@@ -0,0 +1,12 @@
+---
+title: Design
+title-link: https://www.freepik.com/free-vector/design-community-concept-illustration_6610841.htm
```
Comment (SAUMILDHANKAR) [inline _data/internal/credits/design-2.yml:3]: Please use the following correct link: https://thenounproject.com/icon/design-3160558/

File: assets/images/infinity/noun-infinity-686117.svg [hunk 35]
```diff
@@ -0,0 +1,116 @@
...
+  <use x="331.011719" y="700" xlink:href="#e"/>
+  <use x="368.117188" y="700" xlink:href="#o"/>
+  <use x="393.765625" y="700" xlink:href="#d"/>
+  <use x="411.027344" y="700" xlink:href="#b"/>
+  <use x="435.074219" y="700" xlink:href="#n"/>
+  <use x="447.070312" y="700" xlink:href="#a"/>
+  <use x="470.808594" y="700" xlink:href="#m"/>
+  <use x="491.554687" y="700" xlink:href="#c"/>
+ </g>
+</svg>
```
Comment (SAUMILDHANKAR) [inline assets/images/infinity/noun-infinity-686117.svg:116]: We don't need to add this file as it is a duplicate of the following: [assets/images/hero/infinite.svg](https://github.com/hackforla/website/pull/2373/files#diff-f445b1436dff9828d448e19bb8fca38894f203cfda3b92a580341d59be18c3d1) and is already there in the feature branch.

File: _data/internal/credits/justice.yml [hunk 36]
```diff
@@ -0,0 +1,12 @@
+---
+title: Justice
+used-in: Homepage
+artist: Sean Lee
+provider: Unsplash
+title-link: https://unsplash.com/photos/bvHcG6334dE
+content: icon
+provider-link: https://unsplash.com/
+image-url: /assets/images/justice/sean-lee-bvHcG6334dE-unsplash.jpg
```
Comment (SAUMILDHANKAR) [inline _data/internal/credits/justice.yml:9]: As the image already exists in the feature branch, so use that image-url here instead.

File: _data/internal/credits/civictechinfrastructure.yml [hunk 37]
```diff
@@ -0,0 +1,12 @@
+---
+title: Civic Tech Infrastructure
+title-link: https://unsplash.com/photos/unRkg2jH1j0
+content: icon
+used-in: Homepage
+artist: Adeolu Eletu
+provider: Unsplash
+provider-link: https://unsplash.com/
+image-url: /assets/images/civictechinfrastructure/adeolu-eletu-unRkg2jH1j0-unsplash.jpg
```
Comment (SAUMILDHANKAR) [inline _data/internal/credits/civictechinfrastructure.yml:9]: Please change the url with the following url: [assets/images/program-areas/civic-tech-infrastructure.png](https://github.com/hackforla/website/pull/2373/files#diff-d7eb5be6267d0ba2f752538b0ea04a331aa1436b6a3d0aee372b5dc4b3952844) as the image already exists in the feature branch. Also, delete the following file: [assets/images/civictechinfrastructure/adeolu-eletu-unRkg2jH1j0-unsplash.jpg](https://github.com/hackforla/website/pull/2774/files#diff-931215b7d5acfd13ad2ce80f40758494487a403fbce1e5ccf76cc306bb8ddc35)

**Review by SAUMILDHANKAR — CHANGES_REQUESTED** [review]
@jqg123 Thanks for making the changes in your latest commit. We are getting there. Request you to make the following changes:
- The following jpg file used as image for workforce development should not be there as part of this PR, so please remove it. [assets/mages/WorkforceDevelopment/christina-wocintechchat-com-mxOdUd1BQIM-unsplash.jpg]
- The following two files should not be in the files changes tab as it is already there in the feature branch. [_data/internal/credits/operations.yml], [assets/images/operations/noun-operations-1819014.svg](https://github.com/hackforla/website/pull/2774/files#diff-b0c05d9fcef87ca26f5cf0884ef8f879e6c882bcc3cf743e9c64e5519582a0db)(https://github.com/hackforla/website/pull/2774/files#diff-db5a867ab8ddb97cf786803bf4adfc17bb74de24680fb3c2c549bd1a9dc31399), also please do the same for data-science and marketing as well.
- Your accounting yml file should make changes to the revenue yml file which already exists in the feature branch and accounting image is also there with the name of fundraising.
- Your coding yml file should make changes to the engineering yml file which already exists in the feature branch, please make changes to the respective images as well.
- Your design-2 yml file should make changes to the ui-ux yml file which already exists in the feature branch, please make changes to the respective images as well.
- Citizen Engagement yml file should not be in the files changes tab. Changes that you have made to the citizen engagement file should actually be made to citizen-engagement-2 yml file. Also, the image url in citizen-engagement02 file should be [assets/images/program-areas/citizen-engagement.png](https://github.com/hackforla/website/pull/2373/files#diff-c039d78059b82346b5b6f0f0a09781a8c4866bf492c913cdf558affde81d0fbe) as the file already exists in the feature branch.

There are some more changes that need to be made which are similar to those I have already requested, i.e., images for Diversity equity inclusion, social safety net, vote representation, justice, environment are already there in the feature branch so no need to add them here. I will wait for you to make these changes and then take a look again. Please request a re-review once you are done. Thank you.

**Comment by jqg123** [comment]
I may need someone to take over this issue. 
ETA:n/a
Availability: 1hr

**Comment by JessicaLucindaCheng** [comment]
@jqg123
> I may need someone to take over this issue. ETA:n/a Availability: 1hr

Do you want us to un-assign you from the issue related to this pr?

**Comment by SAUMILDHANKAR** [comment]
@jqg123 Just wanted to follow up and know more about the reasons you wanted someone to take over this issue. Do you have any questions, need help or not have enough time or if you would like to work on some other issue instead of this? Please let us know. We will wait for a few days for your response before taking next steps. I will leave a slack message about the same as well. Thank you.

**Comment by SAUMILDHANKAR** [comment]
Received an update from John: "not able to continue on this due to an upcoming job". 
Planning to close this PR since it would take more time to make changes than doing it from scratch. Also, the issue needs a rewrite as well. Including Jessica and John for reference (@jqg123, @JessicaLucindaCheng)

**Pull request closed without merging**